.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+  "printWidth": 120,
```

Comment:
Is this something important respectively do you have a reasoning for introducing this?

Reply:
(I would just try to keep things minimal when possible, but if you have a good reason go for it)

Reply:
I like to keep this as big as possible. You can clearly see benefits when working with TSX and I just try to normalize on a little bit longer lines.

Definitely, I believe that using `80` in 2018 is really unreasonable since we don't code in terminals anymore 😆 

That being said if you're more comfortable with different setting lets change it, I would just urge everyone to try it ;)